main.py

```diff
@@ -1,3 +1,8 @@
-a = 50
-b = 25
-print(a+b)
+spisok = ["apple", "banana", 5, 12, 44, 4.8, True, "Dota", "GtaV", False, 55, 6]
+spisok.remove(5)
+spisok.remove(12)
+spisok.remove(44)
+spisok.remove(4.8)
+spisok.remove(55)
+spisok.remove(6)
+print(spisok)
```

Comment:
This is not the solution to the problem. Try resolve it through the `for loop` and `if`